clippy_utils/src/msrvs.rs

```diff
@@ -12,7 +12,7 @@ macro_rules! msrv_aliases {
 
 // names may refer to stabilized feature flags or library items
 msrv_aliases! {
-    1,62,0 { BOOL_THEN_SOME }
+    1,62,0 { BOOL_THEN_SOME, UNNECESSARY_RESERVE }
```

Comment:
This should be named after the feature that was introduced rather than after the lint.